1299.replace-elements-with-greatest-element-on-right-side.js

```diff
@@ -0,0 +1,19 @@
+/* URL of this problem
+ *  https://leetcode.com/problems/replace-elements-with-greatest-element-on-right-side/
+ *
+ * @param {number[]} arr
+ * @return {number[]}
+ */
+ 
+var replaceElements = function(arr) {
+    const outputArr = [...arr];
```

Comment:
```suggestion
    const outputArr = [];
```
You can also use an empty [] here and push to it